back-end/www/models/model_operations/user_operations.py

```diff
@@ -129,3 +129,61 @@ def remove_user(user_id):
 
     db.session.delete(user)
     db.session.commit()
+
+
+def get_user_count():
+    """
+    Get total user count.
+
+    Returns
+    -------
+    count : int
+        Number of total users.
+    """
+    count = User.query.count()
+
+    return count
+
+
+def get_user_done_location_count(client_id):
+    """
+    Get the distinct location count that the user has ever identified successfully.
+
+    Parameters
+    ----------
+    client_id : str
+        ID provided by an external authentication service.
+
+    Returns
+    -------
+    count : int
+        Number of factories that the user identified.
+    """
+    user = get_user_by_client_id(client_id)
+
+    if user is None:
+        raise Exception("Cannot find the user.")
+
+    if user.answers is None:
+        return 0
+
+    user_answer_list = user.answers
+    identified_answers = {}
+
+    # Exlore all the answers this user reported
+    for user_answer in user_answer_list:
+        # If the location has been marked identified by this user once successfully, skip the matching.
+        if user_answer.location_id in identified_answers.keys():
+            next
```

Comment:
This should be “continue” but not “next”, right?

Reply:
Yes, this should be "continue". My mistake.
However I need to re-write this function, because I mistook the answer submit process before last meeting,  that's why it looks so complicated. It should be very simple as you said.